src/compas/datastructures/halfedge/halfedge.py

```diff
@@ -300,36 +275,48 @@ def face_sample(self, size=1):
         list[int]
             The identifiers of the faces.
 
+        See Also
+        --------
+        :meth:`vertex_sample`, :meth:`edge_sample`
+
         """
         return sample(list(self.faces()), size)
 
-    def key_index(self):
-        """Returns a dictionary that maps vertex dictionary keys to the
+    def vertex_index(self):
+        """Returns a dictionary that maps vertex identifiers to the
         corresponding index in a vertex list or array.
 
         Returns
         -------
         dict[int, int]
-            A dictionary of key-index pairs.
+            A dictionary of vertex-index pairs.
+
+        See Also
+        --------
+        :meth:`index_vertex`
 
         """
         return {key: index for index, key in enumerate(self.vertices())}
 
-    vertex_index = key_index
+    key_index = vertex_index
```

Comment:
Interesting trick ;) 
is this really necessary?/ gets properly documented?

Reply:
no indeed, we could just remove them